src/themes/website/styles/docs.scss

```diff
@@ -214,15 +212,13 @@ validation-summary {
     display: block;
 }
 
-.consoles {
-    background: $panel-bg-color-highlight;
-    height: 100%;
+operation-console {
```

Comment:
`.consoles` class was introduced to define the common styles from both `operation-console` and `graphql-console`. 
Also, if I remember correctly, removing it will re-introduce the bug with the height of the graphql-console, as the background for the console was defined here.

Why would we remove it now?

Reply:
Good catch! That was messed up merge from managed repo. Fixed it.